test/query_test.js

```diff
@@ -1243,6 +1243,18 @@ describe('query', function () {
     return Promise.all([p1, p2]);
   });
 
+  it('to_double', function () {
+    var p1 = assertQuery(query.ToDouble(3.14), 3.14);
+    var p2 = assertQuery(query.ToDouble("3.14"), 3.14);
+    return Promise.all([p1, p2]);
+  });
+
+  it('to_integer', function () {
+    var p1 = assertQuery(query.ToInteger(10.0), 10);
```

Comment:
you could wrap you function on a `ToString(...)` function, that way you can assert that the output string doesn't have a fraction, like
```javascript
assertQuery(query.ToString(query.ToInteger(10.0)), "10");
```